### PR TITLE
nsc-events-nextjs_7_295_arrow_icons

### PR DESCRIPTION
**Pull request by nwm516**
(Help is needed on this User Story/Work in progress)

Relates to #293 

- [ ] Add arrow icons to the left and right side of the event.

When the right arrow is clicked, the event being displayed switches to the next one. If there are no more events, the right arrow should disappear.

When the left arrow is clicked, the event being displayed switches to the previous one. If there are no more previous events, the left arrow should disappear.

Image

### CODE REVIEW

**Comment by nwm516** [comment]
I need to hand off this User Story to another capable developer. This is beyond my scope of knowledge, so any assistance is greatly appreciated. Thank you.

**Review by Robel-003 — APPROVED** [review]
Looks good. I'm able to see the dates in the proper format mm-dd-yyyy format without the timestamp. I'm also able to see the arrow on the event-detail page but when I click the arrow it doesn't change the event. I can pick up where you left off on this user story/issue

**Comment by Diego-Cano** [comment]
Hello i think this will be a great imporvement for the UI on the app.  I agree that the left arrow should disappear if there are no previous events, and the right arrow should do the same if there are no more upcoming events. The changes should be made in the page.tsx file.

I noticed that clicking the arrows doesn't change the event being displayed yet, so that part still needs fixing. On the plus side, the date format issue is sorted, and dates now show up correctly without the timestamp, Good Job. This improvement makes the event details page look much better.

For more info on navigation features in React I found this link : https://reactrouter.com/en/main/components/navigate

File: app/event-detail/page.tsx
Comment (gitbiruk2010) [inline app/event-detail/page.tsx:1]: The addition of navigation arrows enhances user experience by providing intuitive navigation between events. Great job on improving the interface and making it more user-friendly!

**Comment by brinkbrink** [comment]
> Looks good. I'm able to see the dates in the proper format mm-dd-yyyy format without the timestamp. I'm also able to see the arrow on the event-detail page but when I click the arrow it doesn't change the event. I can pick up where you left off on this user story/issue

Thank you, Robel!

**Review by nickolasram — APPROVED** [review]
Everything seems to work well for me. Great job! 👌

**Comment by tinpham5614** [comment]
There are conflicts with the main branch.

**Review by taylorpapke — APPROVED** [review]
Looks good on my end:
Screenshot 2024-05-26 at 2 00 34 PM

**Review by intisarosman1 — APPROVED** [review]
The date format is looking good and so are the arrow icons so far!

**Review by heosman — APPROVED** [review]
The arrow buttons look great and I'm able to click them. Make sure to resolve the conflicts so this PR can be merged asap @nwm516

**Comment by MaddasaHatter** [comment]
So far it looks good the function is there and I can really see a difference in usability with these additions. keep up the good work. I did notice though there may be some merge conflicts.

**Comment by kaimanasse15** [comment]
image
The arrows looks good but I am not able to create an event but I think some data is conflicinting and not your error.

**Pull request merged**